### PR TITLE
Ship command

### LINKED ISSUE #535: Ship command from Baba

### PR DESCRIPTION
**Pull request by metehus**
## Ship command
Closes #535 
`s!ship <user1> [user2]`

### TODOS:
- [x] Fix command usage and descroiption
- [x] Mesasges

### CODE REVIEW

**Comment by metehus** [comment]
Please commit your messages

**Comment by dpaiv0** [comment]
merdacassi

File: src/locales/en-US/commands.json [hunk 1]
```diff
@@ -921,5 +921,48 @@
...
+        "4": "{{shipName}}?!!? OwO",
+        "5": "\"I accept that\" - a random switchblade developer",
+        "6": "{{firstName}} don't be scared and DM {{secondName}}"
+      },
+      "medium": {
+        "0": "{{firstName}} could try but i don't think that {{secondName}} would want",
+        "1": "{{percent}}%... well... there are more people in the world you know...",
+        "2": "{{firstName}} and {{secondName}} friends, but maybe....",
+        "3": "{{shipName}}: I approve!",
+        "4": "{{secondName}} do you aprove that?",
```
Comment (LogGame-HU-DEaN) [inline src/locales/en-US/commands.json]: approve*

File: src/locales/en-US/commands.json [hunk 2]
```diff
@@ -921,5 +921,48 @@
...
+        "5": "Nothing is impossible!",
+        "6": "{{firstName}} and {{secondName}} seem to be only knows"
+      },
+      "low": {
+        "0": "{{shipName}}? are you serious?",
+        "1": "{{shipName}}? i dont think {{firstName}} will like it",
```
Comment (LogGame-HU-DEaN) [inline src/locales/en-US/commands.json]: I don't*

File: src/locales/en-US/commands.json [hunk 3]
```diff
@@ -921,5 +921,48 @@
...
+        "2": "Nothing is impossible.... but this is",
+        "3": "owo whats this? {{shipName}}?? owo im out",
```
Comment (LogGame-HU-DEaN) [inline src/locales/en-US/commands.json]: I'm*

File: src/locales/en-US/commands.json [hunk 4]
```diff
@@ -921,5 +921,48 @@
...
+        "4": "{{secondName}} may not accept it, but i'm always here supporting you :)",
```
Comment (LogGame-HU-DEaN) [inline src/locales/en-US/commands.json]: I'm*

File: src/locales/en-US/commands.json [hunk 5]
```diff
@@ -921,5 +921,48 @@
...
+        "5": "{{shipName}}? WHY?",
+        "6": "i think neither {{firstName}} and {{secondName}} liked it"
+      },
+      "self": {
+        "0": "Well, self love is everything",
+        "1": "You may kiss yourself",
+        "2": "\"I wish i had that confidence\" - a random switchblade developer",
```
Comment (LogGame-HU-DEaN) [inline src/locales/en-US/commands.json]: I*

File: src/locales/en-US/commands.json [hunk 6]
```diff
@@ -921,5 +921,48 @@
...
+        "3": "...wait what?",
+        "4": "Lots of love for {{firstName}} and.... {{secondName}}?",
+        "5": "Self love!! thats the spirit!!",
```
Comment (LogGame-HU-DEaN) [inline src/locales/en-US/commands.json]: That's*

File: src/locales/en-US/commands.json (hunk 3 above)
Comment (LogGame-HU-DEaN) [inline src/locales/en-US/commands.json]: what's*

File: src/locales/en-US/commands.json [hunk 7]
```diff
@@ -921,5 +921,48 @@
...
+  },
+  "ship": {
+    "commandDescription": "Ship you or someone with someone else.",
+    "commandUsage": "<first user> [second user]",
+    "noUser": "Please insert someone to ship",
+    "messages": {
+      "high": {
+        "0": "the image says {{percent}}% but i say 200%",
+        "1": "I always knew about {{shipName}}",
+        "2": "{{shipName}}?!!? thats cute :3",
```
Comment (LogGame-HU-DEaN) [inline src/locales/en-US/commands.json]: That's*

File: src/locales/en-US/commands.json [hunk 8]
```diff
@@ -921,5 +921,48 @@
...
+    "commandUsage": "<first user> [second user]",
+    "noUser": "Please insert someone to ship",
+    "messages": {
+      "high": {
+        "0": "The image says {{percent}}%, but I say 200%",
+        "1": "I always knew about {{shipName}}",
+        "2": "{{shipName}}?!!? That's cute :3",
+        "3": "{{firstName}} you may kiss {{secondName}}",
+        "4": "{{shipName}}?!!? OwO",
+        "5": "\"I accept that\" - a random switchblade developer",
```
Comment (dpaiv0) [inline src/locales/en-US/commands.json]: `switchblade` should be capitalised - `Switchblade`

File: src/locales/en-US/commands.json [hunk 9]
```diff
@@ -921,5 +921,48 @@
...
+        "6": "{{firstName}} don't be scared and DM {{secondName}}"
+      },
+      "medium": {
+        "0": "{{firstName}} could try but I don't think {{secondName}} would want that",
+        "1": "{{percent}}%... well... there are more people in the world you know...",
+        "2": "{{firstName}} and {{secondName}} friends, but maybe...",
```
Comment (dpaiv0) [inline src/locales/en-US/commands.json]: `"{{firstName}} and {{secondName}} could be friends, but maybe..."`

File: src/locales/en-US/commands.json [hunk 10]
```diff
@@ -921,5 +921,48 @@
...
+        "3": "{{shipName}}: I approve!",
+        "4": "{{secondName}} do you approve this?",
```
Comment (dpaiv0) [inline src/locales/en-US/commands.json]: `"{{secondName}}, do you approve this?"`

File: src/locales/en-US/commands.json [hunk 11]
```diff
@@ -921,5 +921,48 @@
...
+        "5": "Nothing is impossible!",
+        "6": "Seems like {{firstName}} and {{secondName}} are just strangers to each other"
+      },
+      "low": {
+        "0": "{{shipName}}? are you serious?",
+        "1": "{{shipName}}? i dont think {{firstName}} will like it",
```
Comment (dpaiv0) [inline src/locales/en-US/commands.json]: `"{{shipName}}? I don't think {{firstName}} will like it"`

**Pull request merged**